Dockerfile.local

```diff
@@ -9,13 +9,15 @@ WORKDIR /app
 
 COPY .build/dependencies-state.json ./.build/dependencies-state.json
 COPY .env ./
-COPY Package.resolved ./
 COPY Packages ./Packages
 
+COPY Makefile ./
+RUN make install-cmark
+
 COPY Package.swift ./
-RUN swift package update
+RUN OSS=1 swift package update
```

Comment:
forced to use `OSS=1` because the linux container cant close the private repo...